core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-properties/dot-container-properties.component.ts

```diff
@@ -59,16 +54,46 @@ export class DotContainerPropertiesComponent implements OnInit {
                     preLoop: container?.preLoop ?? '',
                     postLoop: container?.postLoop ?? '',
                     containerStructures: this.fb.array(
-                        containerStructures ?? [],
+                        [],
                         containerStructures.length ? [Validators.minLength(1)] : null
                     )
                 });
+
+                this.addContainerFormControl(containerStructures);
             });
         this.form.valueChanges.pipe(takeUntil(this.destroy$)).subscribe((values) => {
             this.store.updateIsContentTypeButtonEnabled(values.maxContentlets > 0);
         });
     }
 
+    /**
+     * Add Container Strcutures into FormControl
+     * @param {DotContainerStructure[]} containerStructures
+     * @memberof DotContainerPropertiesComponent
+     */
+    addContainerFormControl(containerStructures: DotContainerStructure[]) {
+        if (containerStructures && containerStructures.length > 0) {
+            containerStructures.forEach(
+                ({ code, structureId, containerId, containerInode, contentTypeVar }) => {
+                    (this.form.get('containerStructures') as FormArray).push(
+                        this.fb.group({
+                            code: new FormControl(code, [Validators.required]),
+                            structureId: new FormControl(structureId, [Validators.required]),
+                            containerId: new FormControl(containerId),
+                            containerInode: new FormControl(containerInode),
+                            contentTypeVar: new FormControl(contentTypeVar)
+                        })
+                    );
+                }
+            );
+
+            // Using setTimeout because dotGlobal Message Service throwing ExpressionChangedAfterItHasBeenCheckedError Error
+            setTimeout(() => {
+                this.showContentTypeAndCode();
```

Comment:
Can you please check if why we need to call the `dotGlobalMessageService` inside the store?

Reply:
I also updated you about this on slack. basically It is showing a message when it is getting content Types from API but I don't think it is necessary.
![image](https://user-images.githubusercontent.com/113915849/204489746-e633395e-051e-48c5-af22-9fbe7b26b67b.png)

Reply:
updated